Regular-Question-Answers/n-ary-tree-level-order-traversal.cpp

```diff
@@ -0,0 +1,38 @@
+// Author: Murat Biberoğlu
+// Question Link: https://leetcode.com/problems/n-ary-tree-level-order-traversal/
+
+class Solution {
+   public:
+    // we are going to us breadth first search algorithm (bfs) to solve this
+    // question, because we need to get nodes consecutive that have the same level.
+    vector<vector<int>> levelOrder(Node* root) {
+        vector<vector<int>> levels;
+        // if there is no node, then return empty vector
+        if (!root) {
+            return levels;
+        }
+
+        // Basic Iterative BFS
+        queue<Node*> q;
+        q.push(root);
+        while (!q.empty()) {
+            // get the size of the queue before adding new childs.
+            // in that state queue has nodes that have same level.
+            int levelSize = q.size();
+            // create a vector with size as current level's node count
```

Comment:
Correct to "vector with size **the same** as current level's node count" or "vector with size **of the** current level's node count"